build/index.html

```diff
@@ -47,6 +47,16 @@ <h2 id="content-9">Skipulag námsins í HR</h2>
             <p id="content-10">Um er að ræða tveggja ára, þverfaglegt, 120 eininga nám. Einingarnar skiptast í 44-78 einingar úr námskeiðum á meistarastigi (kennd í HR og HÍ), 0-30 einingar úr grunnnámskeiðum í tölvunarfræði, 0-10 einingar úr grunnnámskeiðum í málfræði (kennd í HÍ) og 30-60 einingar í meistaraprófsverkefni/ritgerð. Samsetning eininga er því  sveigjanleg og fer eftir bakgrunni viðkomandi nemanda.</p>
             <p id="content-11">Nemendur með BA-próf í málvísindum og tungumálum þurfa að taka 30 einingar í grunnnámskeiðum í tölvunarfræði. Þessi námskeið eru metin sem hluti meistaranámsins. Nemendur með aðra undirstöðu gætu þurft að taka grunnnámskeið í bæði málfræði og tölvunarfræði.</p>
         </div>
+        <footer class="footer mt-auto bg-dark">
+            <nav class="navbar">
+                <p class="navbar-nav ml-auto">
+                    <a class="nav-link text-white" href="solskin.html" >Dæmi 2
+                    <svg xmlns="http://www.w3.org/2000/svg" width="25" height="25" fill="currentColor" class="text-white bi bi-arrow-right" viewBox="0 0 16 16">
```

Comment:
Changes look good, but perhaps it would be better for these two pages to share the svg? I.e. as an img instead of an inline element.

Reply:
now it is a shared image but since the svg is black, I've added some css to make it white. it's not in the same css file as the rest because this is not technically webrice css, just css for the demos.